### PR TITLE
Omniture: is Facebook or Twitter app 

### PR DESCRIPTION
**Pull request by stephanfowler**
Tell Omniture if we're within an in-app Facebook or Twitter browser.

### CODE REVIEW

File: static/src/javascripts/projects/common/utils/detect.js [hunk 1]
```diff
@@ -133,6 +133,14 @@ define([
         return navigator.mozApps && !window.locationbar.visible;
     }
 
+    function isFacebookApp() {
+        return navigator.userAgent.indexOf('FBAN/') > -1;
+    }
+
+    function isTwitterApp() {
+        return navigator.userAgent.indexOf('Twitter for iPhone') > -1;
```
Comment (robertberry) [inline static/src/javascripts/projects/common/utils/detect.js]: What about on Android?

**Comment by robertberry** [comment]
:+1: otherwise

File: static/src/javascripts/projects/common/utils/detect.js (hunk 1 above)
Comment (stephanfowler) [inline static/src/javascripts/projects/common/utils/detect.js]: Seems you [can't distinguish the Android Twitter app from Chrome](http://mobiforge.com/research-analysis/webviews-and-user-agent-strings)

Comment (robertberry) [inline static/src/javascripts/projects/common/utils/detect.js]: Adding a comment to that effect will help future users reading your code :dancer:

**Comment by robertberry** [comment]
SHIPPIT

**Pull request merged**